docker/Dockerfile

```diff
@@ -29,11 +30,12 @@ RUN apk add --no-cache librsvg ttf-opensans
 WORKDIR /invidious
 RUN addgroup -g 1000 -S invidious && \
     adduser -u 1000 -S invidious -G invidious
-COPY ./assets/ ./assets/
-COPY ./config/config.yml ./config/config.yml
-COPY ./config/sql/ ./config/sql/
-COPY ./locales/ ./locales/
+COPY --chown=invidious ./assets/ ./assets/
```

Comment:
Why do all those files need to owned by `invidious`?

Reply:
I think it is because if the file is owned by root, the invidious user might not be able to read it. and the docker will fail.

Reply:
e.g. if only owner can read